spooky-oysters/bangazon
# Gl refactor models

## Link to Ticket

no direct ticket to reference

## Description of Proposed Changes

- Refactored models to switch from IEnumerable collections to ICollections set to public virtual with a get/set
- Performed a database migration

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout gl-refactorModels
```
Delete your database file
Perform a database update and start server
```sh
dotnet ef database update
dotnet run
```

Check models in code editor and make sure the ones that are not joiner tables, have a ICollection set to `public virtual` and not an IEnumerable. 

## Impacted Areas in Application

List general components of the application that this PR will affect:

* models
* migration folder
* database


## Mentions @username

Tag users that need to review this code
